owners/src/github.js

```diff
@@ -139,10 +139,10 @@ class GitHub {
    *     https://probot.github.io/api/latest/interfaces/githubapi.html).
    * @param {string} owner GitHub repository owner.
    * @param {string} repository GitHub repository name.
-   * @param {!Logger} logger logging interface.
+   * @param {Logger=} [logger=console] logging interface.
```

Comment:
Same comment as the other PR. It's not common in `amphtml` to use JSDoc to specify default args. Should the function signature be as follows?
```
constructor(client, owner, repository, logger=console)
```

Reply:
See #568